Core/clim-core/frames.lisp

```diff
@@ -725,6 +728,29 @@ documentation produced by presentations.")
 ;
 ; FIXME
 
+(defun update-frame-pane-lists (frame)
+  (let ((all-panes     (frame-panes frame))
+        (named-panes   (mapcar #'cdr (frame-panes-for-layout frame)))
+        (current-panes '()))
+    ;; Find intersection of named panes and current layout panes.
+    (map-over-sheets (lambda (sheet)
+                       (when-let ((index (position sheet named-panes)))
+                         (push (cons sheet index) current-panes)))
+                     all-panes)
+    (setf current-panes (map 'list #'car (sort current-panes #'< :key #'cdr)))
```

Comment:
mapcar #'car ?

Reply:
please use list operators where we deal with lists

Reply:
Changed.